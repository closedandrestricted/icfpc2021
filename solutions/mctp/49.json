{"vertices":[[1,28],[15,19],[19,22],[8,44],[20,43],[18,57],[14,3],[29,54],[29,59],[17,54],[21,39],[9,44],[22,1]]}