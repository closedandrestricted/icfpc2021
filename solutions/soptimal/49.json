{"vertices":[[11,11],[2,24],[0,29],[7,44],[16,53],[31,50],[12,0],[30,56],[30,61],[18,58],[21,43],[10,45],[22,0]]}