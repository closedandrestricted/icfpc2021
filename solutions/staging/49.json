{"vertices":[[1,27],[16,21],[19,17],[7,44],[20,46],[30,59],[22,0],[24,59],[19,57],[31,50],[17,46],[7,40],[12,0]]}