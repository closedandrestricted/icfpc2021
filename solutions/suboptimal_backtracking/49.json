{"vertices":[[16,19],[2,25],[1,30],[7,41],[18,44],[31,50],[22,0],[21,58],[26,60],[15,54],[21,41],[9,39],[12,0]]}