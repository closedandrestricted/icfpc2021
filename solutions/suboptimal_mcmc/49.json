{"vertices":[[11,10],[3,23],[1,28],[10,39],[16,49],[31,50],[22,0],[27,57],[30,61],[18,58],[19,44],[7,44],[12,0]]}