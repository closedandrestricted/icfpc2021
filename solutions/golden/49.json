{"vertices":[[19,29],[19,45],[24,47],[5,26],[16,19],[10,5],[30,61],[16,5],[21,3],[9,8],[17,22],[6,27],[28,60]]}